meteor/server/migration/currentSystemVersion.ts

```diff
@@ -49,4 +49,4 @@
  */
 
 // Note: Only set this to release versions, (ie X.Y.Z), not pre-releases (ie X.Y.Z-0-pre-release)
-export const CURRENT_SYSTEM_VERSION = '1.46.0'
+export const CURRENT_SYSTEM_VERSION = '1.47.0'
```

Comment:
TODO: ensure that this diff goes away during merge (or when updating the branch with the latest release-branch)